README.md

```diff
@@ -93,4 +93,39 @@ http://127.0.0.1:TUNNEL_PORT/v1beta1/namespaces/1/datasets
 
 ## License
 
+### Test AUTOML java web demo server
+
+Load local image to minikube
```

Comment:
What’s the execution order of commands in the Readme? Looks like the image load command should be prepend to the kubectl apply command. And we also need to add build docker image command before image load command.

Reply:
I moved the image load command to the Start Deploy part. And how to build the java docker image, we can find the doc in java project

Reply:
Not all contributors in our project are software engineers. There are perhaps also algorithm engineers and UI designers as well as product managers. All commands need to be run to deploy AutoML platform should be put into readme doc so that they can install the platform easily.